src/pow.cpp

```diff
@@ -147,6 +219,41 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
     return bnNew.GetCompact();
 }
 
+/** Perform the fork difficulty reset */
+unsigned int CalculateMVFResetWorkRequired(const CBlockIndex* pindexLast, int64_t nFirstBlockTime, const Consensus::Params& params)
+{
+    const arith_uint256 bnPowLimit = UintToArith256(params.powLimit); // MVF-BU moved here
+
+    arith_uint256 bnNew, bnNew1, bnNew2, bnOld;
+
+    // drop difficulty via factor
+    int nDropFactor = 4;
```

Comment:
suggest to get rid of hardcoded factor by creating a parameter in consensus params which is set by constants defined in our MVF header file. This will make it easier to adjust the values per network for tests and release.